src/templates/antora/template.ts

```diff
@@ -197,38 +115,11 @@ const flatten = <T>(array: T[][]): T[] => {
   return ([] as T[]).concat(...array);
 };
 
-/**
- * [compileTemplate description]
- * @param  {[type]} filePath [description]
- * @return {[type]}          [description]
- */
-const compileTemplate = (filePath: string): HandlebarsTemplateDelegate => {
-  return Handlebars.compile(fs.readFileSync(path.join(__dirname, filePath)).toString());
-};
-
-/**
- * [createFileName description]
- * @param  {[type]} data [description]
- * @param  {[type]} ext [description]
- * @return {[type]}          [description]
- */
-const createFileName = (data: Record<string, any>, ext: 'adoc' | 'json'): string => {
-  if ('adoc' === ext) {
-    if (data.fullName === 'tinymce') {
-      return (BASE_PATH + '/tinymce.root.adoc').toLowerCase();
-    }
-
-    return (BASE_PATH + '/' + data.fullName + '.adoc').toLowerCase();
-  } else if ('json' === ext) {
-    return ('_data/api/' + data.type + '_' + data.fullName.replace(/\./g, '_') + '.json').toLowerCase();
-  }
-};
-
 /**
  * [addPageToArchive description]
  * @param {[type]} page [description]
  */
-const addPageToArchive = function (this: ZipWriter, page: { filename: string; content: string }) {
+const addPageToArchive = function (this: ZipWriter, page: PageOutput) {
   this.addData(page.filename, page.content);
```

Comment:
There is a bug in the zip output it produces paths with double `/` in them like `/_data/antora/api/tinymce.html//tinymce.html.writer.adoc` for example. Noticed that one of the zip tools I have on my machine was adding empty folders in the structure. We should fix this bug even though unzip seems to just ignore that extra slash.
Also you could bring in the Reporter here and do a `Reporter.info(`Adding file to zip: ${page.filename}`);` call here so that one can inspect what it's doing having the reporter could be useful for other things as well in this template.

- [ ] Fix double slash bug and maybe add reporting

Reply:
Reporting added and bug fixed, thanks for finding this!